Jenkinsfile-release

```diff
@@ -30,6 +30,18 @@ pipeline {
             }
         }
 
+        stage('Commit Proto.lock Files') {
+            steps {
+                if ("${version}".endsWith('Final')) {
+                    configFileProvider([configFile(fileId: 'maven-settings-with-deploy-snapshot', variable: 'MAVEN_SETTINGS')]) {
+                        sh "$MAVEN_HOME/bin/mvn clean install -B -V -e -s $MAVEN_SETTINGS -DskipTests -DcommitProtoLockChanges"
```

Comment:
Shouldn't this be `-DcommitProtoLockChanges=true`?

Reply:
Simply setting `-DcommitProtoLockChanges` was sufficient in my trials, however I'll add `=true` to make it more explicit. This behaviour could also be a consequence of not explicitly setting a default of false for `${commitProtoLockChanges}`